### DISCUSSION TITLE
I didn't recieve my student pack after 10 days

### DISCUSSION THREAD

**Discussion by ahmetkutay** [community/community#147216] [GitHub Education]
### Hi everyone,

As the title says, I'm waiting for my student developer pack, but there is no news for me. It was approved 10 days ago, 6 Dec 2024.

**Comment by samus-aran** [comment]
Hi there! 👋 

That's not normal 😅 I've raised this with the internal team and we'll comment on this thread when there is an update 👍

**Reply by ahmetkutay** [reply]
Hello, many thanks. I'll be waiting.

**Reply by ahmetkutay** [reply]
> Hi there! 👋 
> 
> 
> 
> That's not normal 😅 I've raised this with the internal team and we'll comment on this thread when there is an update 👍 

Hi again, As I expected, I am rejected because of adding wrong university, we can close this discussion, I reapply for my other university. Thanks
